images/runtime/php-fpm/.gitignore

```diff
@@ -0,0 +1,2 @@
+Dockerfile
+base.Dockerfile
```

Comment:
do we need this file? doesn't the root .gitignore contain similar entries?

Reply:
yeah I will remove it ..